js/kucoin.js

```diff
@@ -513,9 +509,7 @@ module.exports = class kucoin extends Exchange {
         const marketId = market['id'];
         const request = {
             'symbol': marketId,
-            'startAt': Math.floor (since / 1000),
             'endAt': this.seconds (),
-            'pageSize': limit,
```

Comment:
Cut this one out of here correctly, but we still have to restore and handle the case of a defined `limit`:

```JavaScript
const request = { 
    ... 
};
if (limit !== undefined) {
    request['pageSize'] = limit;
}
```

Reply:
Oops, saw this in further commits, so nvm ;)